bin/terminus.php

```diff
@@ -30,6 +30,12 @@
 $output = new ConsoleOutput();
 Robo::configureContainer($container, $input, $output, $application);
 
+$container->share('fileCache', \Terminus\Caches\FileCache::class);
+$container->share('session', \Pantheon\Terminus\Session\Session::class)
+    ->withArgument('fileCache');
+$container->inflector(\Pantheon\Terminus\Session\SessionAwareInterface::class)
```

Comment:
Question! Why do we need to share and inflect these while `OutputAwareInterface`, `InputAwareInterface`, and `LoggerAwareInterface` all don't need it?